catalog/additionalInfo.md

```diff
@@ -0,0 +1,3 @@
+## Additional Info
```

Comment:
This file should be removed, as this placeholder content will display if it's here.